packages/design-system/src/icons/svgs/third-party-brands/brand-nftgeek-24.icon.ts

```diff
@@ -0,0 +1,57 @@
+import {html} from 'element-vir';
+import {defineIcon} from 'vira';
+import {toniqIconCssVarWithFallback} from '../../../styles';
+
+export const BrandNftgeek24Icon = defineIcon({
+    name: 'BrandNftgeek24Icon',
+    svgTemplate: html`
+        <svg
```

Comment:
this NFTGeek logo is outdated. Here's the new one:
![nft-geek](https://github.com/Toniq-Labs/toniq-labs-design-system/assets/1205860/5661b194-8116-418c-bc7f-7511c8f81029)

and the SVG for it:
```html

        <svg
            xmlns="http://www.w3.org/2000/svg"
            stroke="none"
            width="24"
            height="24"
            viewBox="0 0 24 24"
            fill=${toniqIconCssVarWithFallback['toniq-icon-fill-color']('#000')}
        >
            <path
                d="M9.75 17.23h3.77v-4.02H10.4v2.03H4.75v-5.7h5.64v2.32h3.13V9.55h5.65v5.7h-4.3v3.33H9.74v-1.35ZM6.1 21.65h11.76v-5.07h1.34V23H4.75v-6.42h1.34v5.07Zm9.74-13.56-.01-.14-.13-3.39v-.15l-.02-.2-.08-1.66-.71 1.32-.05.09-.55 1.02-.12.23-1.48 2.73-.07.15-.01.01H11.49l-.02-.01-.02-.01-.02-.19-.29-3.22-.16.3-1.6 2.97-.08.15H8.15l-.05-.16-.82-2.37-1.29 2.4-.07.13H4.78l.65-1.2L6.7 4.54h1.3l.8 2.35.37-.68 1.6-2.98h1.26l.24 2.62.06.65 1.42-2.63L15.28 1h1.23l.12 2.5.06 1.06.01.31v.03l.06 1.49 1.35-2.5h1.14l-.02.04-.58 1.06-1.6 2.98-.06.12h-1.16Z"
            />
        </svg>
```